esp/src/eclwatch/templates/LockDialogWidget.html

```diff
@@ -9,7 +9,7 @@
             <p id="UnlockStatus"></p>
         </div>
         <div class="dijitDialogPaneActionBar">
-            <button id="${id}UnlockBtn" data-dojo-attach-event="onClick:_onUnlock" data-dojo-type="dijit.form.Button">${i18n.Unlock}</button>
+            <button id="unlock" data-dojo-attach-event="onClick:_onUnlock" data-dojo-type="dijit.form.Button">${i18n.Unlock}</button>
```

Comment:
Is this safe?  I assume we can't have two instances of the lock dialog open at the same time?

Reply:
I use a destroyRecusrive() to eliminate any "already exists" message.